leases/tests/test_lease_mutations.py

```diff
@@ -26,8 +26,10 @@
 from contracts.tests.factories import BerthContractFactory
 from customers.schema import BoatNode, ProfileNode
 from customers.tests.conftest import mocked_response_profile
+from payments.enums import OrderStatus
 from payments.models import Order
 from payments.schema import PlaceProductTaxEnum
+from payments.tests.conftest import *  # noqa
```

Comment:
You should do this import to `leases/tests/conftest.py` rather than directly here

Reply:
fixed